Packs/CommonScripts/ReleaseNotes/1_19_19.md

```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+
+##### CreateNewIndicatorsOnly
+Fixed an issue where the value of existing indicator had different letter case than indicator on input.
```

Comment:
Fixed an issue where the value of an existing indicator had a different letter case than the indicator on input.